src/fastoad/gui/analysis_and_plots.py

```diff
@@ -148,7 +148,10 @@ def aircraft_geometry_plot(
     )
 
     # Horizontal Tail parameters
-    ht_root_chord = variables["data:geometry:horizontal_tail:root:chord"].value[0]
+    if "data:geometry:horizontal_tail:center:chord" not in variables.names():
```

Comment:
I think a comment to state we keep backward compatibility here would be good.

And as a side note, `if x in seq:` is generally preferred than `if x not in seq:`.
I mean, your choice here is driven only by code history. If you had coded that from scratch, no doubt you would have done it the other way.

Reply:
Corrected in [54240a5](https://github.com/fast-aircraft-design/FAST-OAD/pull/546/commits/54240a58b538344a127a200d68c3fe13f00df9f0)